Instastories.py

```diff
@@ -285,8 +285,14 @@ def nick_to_id(nickname):
     Returns:
         ids (number): Id
     """
+    try:
+        cookie = settings.get('session_id')
+    except Exception as e:
```

Comment:
Don't catch all Exception, only KeyException. But even better don't use Exception handling here but just a simple check with has_setting() or use get_settings_file and check yourself so you save a file load.

Reply:
Done